core/domain/config_domain.py

```diff
@@ -208,6 +208,7 @@ def init_config_property(cls, name, instance):
 
         Args:
             name: str. The name of the configuration property.
+            instance: instance. The instance of the configuration property.
```

Comment:
Specify the type of the instance,

Reply:
This function takes in any general instance and initializes the _config_registry. I think that it does not have a specific type. Please suggest. Thanks!

Reply:
`Registry` would be a good type since that's the type of the instance.

Reply:
Done.